### DISCUSSION TITLE
Popover on polls can't be dismissed

### DISCUSSION THREAD

**Discussion by xdumaine** [community/community#34224] [Discussions] [answered]
![bug](https://user-images.githubusercontent.com/833911/192310377-825f9139-ab69-456b-84ed-aa12edc2f748.gif)

**Comment by queenofcorgis** [comment]
Thanks for flagging! I've added the appropriate label to this so our team can investigate.

**Comment by siminapasat** [comment, marked as answer]
This should be fixed now 😊
